docs/source/index.rst

```diff
@@ -166,6 +167,7 @@ Please see our `contributing guidelines <https://github.com/cleanlab/cleanlab/bl
    cleanlab/multilabel_classification/index
    cleanlab/token_classification/index
    cleanlab/object_detection/index
+   cleanlab/segmentation/index
```

Comment:
```suggestion
   cleanlab/segmentation/index
   cleanlab/object_detection/index
```

Reply:
recommend this order because segmentation is very similar to token-classification whereas object detection is completely different